src/pages/trade/ui/chart.tsx

```diff
@@ -119,10 +120,11 @@ const ChartLoadingState = () => {
   );
 };
 
-const ChartData = observer(({ candles }: { candles: OhlcData[] }) => {
+const ChartData = observer(({ candles }: { candles: CandleWithVolume[] }) => {
   const chartElRef = useRef<HTMLDivElement>(null);
   const chartRef = useRef<IChartApi>();
   const seriesRef = useRef<ReturnType<IChartApi['addCandlestickSeries']>>();
+  const volumeSeriesRef = useRef<ReturnType<IChartApi['addHistogramSeries']>>();
```

Comment:
question: curious, do they ever overlap? What would happen?

Reply:
I added the scaling code so they don't overlap, however they can overlap if there is extremely high volume and a big drop in price. I haven't encountered this so far.